ext/TensorKitAMDGPUExt/TensorKitAMDGPUExt.jl

```diff
@@ -0,0 +1,109 @@
+module TensorKitAMDGPUExt
+
+using AMDGPU, AMDGPU.rocBLAS, LinearAlgebra
+using AMDGPU: @allowscalar
+import AMDGPU: rand as rocrand, rand! as rocrand!, randn as rocrandn, randn! as rocrandn!
+
+using TensorKit
+import TensorKit.VectorInterface: scalartype as vi_scalartype
```

Comment:
Is there another `scalartype` that might conflict with the one from `VectorInterface`?